roles/certmanager/tasks/main.yaml

```diff
@@ -13,3 +13,7 @@
     - cert-manager-cainjector
     - cert-manager-webhook
   when: certmanager_wait_for_deployments | bool
+
+- name: Configure Letsencrypt certificates
+  include: letsencrypt.yaml
+  when: certmaner_use_letsencrypt | bool
```

Comment:
:wrench: Typo? `certmaner` -> `certmanager`

Reply:
You're reviewing outdated

Reply:
Not quite sure how I managed that with a fresh tab :sweat_smile: sorry!